_summaries/chen2020a_2.md

```diff
@@ -1,19 +1,50 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: chen2020a
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: kraja99
+score: 7
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Overview
+The authors introduce SimCLR as a framework to leverage contrastive learning for visual representations. By utilizing combined data augmentations, a learnable nonlinear transformation between the representation and the contrastive loss, and specific training parameters (larger batch size and more training steps) the authors show strong performance on self and semi supervised learning on ImageNet.
+
+## SimCLR
+
+![Framework](chen2020a_2a.png)
+
+In general, the framework trys to process two different versions of a data sample and try to maximize a contrastive loss between the parallel outputs. This begins with some series of data augmentation operations to construct to variations of the input data sample x. Representation vectors of x are then retrieved by putting xi and xj through a ResNet, and finally a projection head (MLP) is used to map hi/hj to a space where the contrastive loss function is applied on the parallel outputs.
+
+The authors train by performing a contrastive prediction task over N data points where given one of the two augmentations as specified above, the model must determine the other augmentation from the same initial example across 2N-2 negative examples. The authors use NT-Xent (normalized temperature-scaled cross entropy loss) as follows: 
```

Comment:
I think it's important to note that N is the batch size, which could explain why this architecture benefits more from larger batch sizes.

I think it would be clearest to say that each data point is run through two different augmentation transformations to create a total of 2N points